vhdbuilder/packer/install-dependencies.sh

```diff
@@ -348,6 +348,20 @@ if grep -q "fullgpu" <<< "$FEATURE_FLAGS" && grep -q "gpudaemon" <<< "$FEATURE_F
 fi
 fi
 
+mkdir -p /var/log/azure/Microsoft.Azure.Extensions.CustomScript/events
+
+systemctlEnableAndStart cgroup-memory-telemetry.timer || exit 1
+systemctlEnableAndStart cgroup-memory-telemetry.service || exit 1
+
+CGROUP_VERSION=$(stat -fc %T /sys/fs/cgroup)
+if [ "$CGROUP_VERSION" = "cgroup2fs" ]; then
+  systemctlEnableAndStart cgroup-pressure-telemetry.timer || exit 1
+  systemctlEnableAndStart cgroup-pressure-telemetry.service || exit 1
+fi
```

Comment:
```suggestion
systemctlEnableAndStart cgroup-memory-telemetry.timer || exit 1
systemctlEnableAndStart cgroup-memory-telemetry.service

CGROUP_VERSION=$(stat -fc %T /sys/fs/cgroup)
if [ "$CGROUP_VERSION" = "cgroup2fs" ]; then
  systemctlEnableAndStart cgroup-pressure-telemetry.timer || exit 1
  systemctlEnableAndStart cgroup-pressure-telemetry.service
fi
```

kubepods doesn't exist at build time